packages/editor/src/lib/editor/managers/TextManager.ts

```diff
@@ -72,6 +72,7 @@ export class TextManager {
 			minWidth?: null | number
 			padding: string
 			disableOverflowWrapBreaking?: boolean
+			renderMethod?(): string | Node
```

Comment:
Yeah, we had this discussion in https://github.com/tldraw/tldraw/pull/3051 :)
It's been a while since that PR though hah.

But yeah, I think I like you're revised take on it 👍

Reply:
I did a slight variation of what you suggested (more like in #3051 - the HTML, not a node, since I don't have a node in the rich text world, per se), take a look at → PR https://github.com/tldraw/tldraw/pull/5131